addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -584,6 +584,13 @@ export const editorCommands = {
         }
 
         let target = [...(blocks.size ? blocks : li)];
+        if (blocks.size) {
+            for (const block of blocks) {
+                if (block.style) {
+                    block.style.padding = null;
```

Comment:
Does it work if you pass it `undefined`? I would assume so. It's bad enough that Javascript has an `undefined` value, let's just all forget about `null` where we can... ^^' Wait, or even better, does it work with `delete vlock.style.padding`?

Reply:
undefined and delete didn't work here , according to chatGPT `CSSStyleDeclaration properties are not deletable`, I instead replaced null by just empty string (suggest by chatGPT too)